### PR TITLE
:sparkles: feat: Add footer with correct styling

### PR DESCRIPTION
**Pull request by kahenya-anita**
Footer

### CODE REVIEW

**Comment by kahenya-anita** [comment]
Task linked: [CU-8669fja8v Footer](https://app.clickup.com/t/8669fja8v)

File: Footer/footer.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./footer.css" />
+    <title>Links</title>
+</head>
+
+<body>
+    <div class="links-container">
+        <div class="links-items">
```
Comment (HenryMoringa) [inline Footer/footer.html]: Either use a div with role of list or preferably use a list element instead

File: Footer/footer.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            <div class="link-top">
```
Comment (HenryMoringa) [inline Footer/footer.html]: Change this into a list item

File: Footer/footer.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+                <div class="links"><a href="#">FAQ</a></div>
+                <div class="links"><a href="#">Events</a></div>
```
Comment (HenryMoringa) [inline Footer/footer.html]: Remove unnecessary divs

**Comment by HenryMoringa** [comment]
- Change the name to conform to agreed upon standard

**Review by HenryMoringa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**